app/selectors/custom_status.ts

```diff
@@ -25,6 +29,21 @@ export function makeGetCustomStatus(): (state: GlobalState, userID?: string) =>
     );
 }
 
+export function isCustomStatusExpired(state: GlobalState, customStatus?: UserCustomStatus) {
+    if (!customStatus) {
```

Comment:
does this work on all servers that support custom status?

Reply:
No, it works on those servers which contain the expiry changes.

Reply:
Tested this PR against 5.36 server which has custom status feature but without expiry, and that seemed to be confusing to end user since the user seemed to be able to add custom expiry but on save/"Done", no feedback is returned (looks successful). However, upon checking the custom status, it actually didn't save anything. So this is not backward compatible. Maybe it should behave the same as the Webapp where the custom expiry is not shown when not supported by the server.

PR against 5.36 server - looks able to setup custom expiry
![Screen Shot 2021-07-06 at 6 21 33 PM](https://user-images.githubusercontent.com/5334504/124585166-9fed1c80-de87-11eb-8b6f-1d16b3558e37.png)

PR against 5.36 server - but on save or "Done", even the custom status is not saved.
![Screen Shot 2021-07-06 at 6 21 50 PM](https://user-images.githubusercontent.com/5334504/124585307-babf9100-de87-11eb-8755-72928fd0f498.png)

Reply:
This is now resolved. Thanks @manojmalik20!